gateway/webhook/integration_test.go

```diff
@@ -46,6 +47,17 @@ import (
 	"github.com/rudderlabs/rudder-transformer/go/webhook/testcases"
 )
 
+var webhookVersion string
```

Comment:
I didn't get how are we planning to run this test? Do we want to run the test for both v1 and v2? If so, we could have a table driven test for both v1 and v2 instead of this flag?

Reply:
I don't see anything modified in `tests.yaml` regarding webhookVersion?

Reply:
Currently, webhook integration tests are run manually from rudder-transformer by the integrations team [using this](https://github.com/rudderlabs/rudder-transformer/blob/develop/go/README.md). This PR is only to enhance this with a commandline option so that we can test out webhook version with v2 as well.
cc: @sanpj2292 

Changes to run both the tests with both versions will come in a different PR associated with the [v2 server changes](https://linear.app/rudderstack/issue/INT-2802/stage-1-or-v2-spec-or-server-changes-for-v2-spec).